cmake/modules/FindDiaSDK.cmake

```diff
@@ -69,6 +69,27 @@ if(NOT DiaSDK_ROOT)
 	endif()
 endif()
 
+# Gather up the include directories listed in the include environment variable.
+set(env_includes)
+if(CMAKE_HOST_SYSTEM_NAME STREQUAL "CYGWIN")
+	# INCLUDE is a list of windows style paths.
+	# This needs to be converted to cygwin style paths to feed to CMake.
+	if(CMAKE_CYGPATH AND (NOT "$ENV{INCLUDE}" STREQUAL ""))
+		execute_process(
+			COMMAND "${CMAKE_CYGPATH}" -c $ENV{INCLUDE}
+			OUTPUT_VARIABLE unix_paths
```

Comment:
Shouldn't this write to `env_includes` instead?

Reply:
Looks like there was an update I made that didn't get committed. This makes more sense with the updated `string(REPLACE...)` line